src/main/kotlin/no/nav/rekrutteringsbistand/stillingssokproxy/SearchClient.kt

```diff
@@ -18,22 +18,31 @@ import org.elasticsearch.client.RestClient
 import org.elasticsearch.client.RestHighLevelClient
 import java.io.IOException
 
-fun sok(jsonbody: String, params: Map<String, List<String>>, indeks: String): SokeResultat {
-    val client = getRestHighLevelClient()
+fun sok(jsonbody: String, params: Map<String, List<String>>, indeks: String): ElasticSearchSvar {
     val request = elasticSearchRequest("GET", "$indeks/_search", params, jsonbody)
+    return gjorRequest(request)
+}
+
+fun explain(jsonbody: String, params: Map<String, List<String>>, indeks: String, dokumentnummer: String): ElasticSearchSvar {
+    val request = elasticSearchRequest("GET", "$indeks/_explain/$dokumentnummer", params, jsonbody)
```

Comment:
Ser vi gjør POST mot selve proxyen, som gjør GET til Elastic Search.
Både `_search` og `_explain` støtter POST, så burde vi bare bruke det?
Så blir proxyen mest mulig lik det faktiske kallet mot Elastic Search, og vi slipper å lure på hvorfor vi gjør to forskjellige ting...

Reply:
Helt i starten satte vi den opp med å gjøre søket med POST. Fikk da feilmelding fra ElasticSearch om manglende tilgang, noe vi antok skyldtes at brukeren som benyttes er read only. Når vi så byttet til GET, fungerte det.